lib/saluki-components/src/transforms/host_tags/mod.rs

```diff
@@ -0,0 +1,119 @@
+use std::{
+    num::NonZeroUsize,
+    time::{Duration, Instant},
+};
+
+use async_trait::async_trait;
+use bytesize::ByteSize;
+use memory_accounting::{MemoryBounds, MemoryBoundsBuilder};
+use saluki_config::GenericConfiguration;
+use saluki_context::{ContextResolver, ContextResolverBuilder};
+use saluki_core::{components::transforms::*, topology::interconnect::FixedSizeEventBuffer};
+use saluki_env::helpers::remote_agent::RemoteAgentClient;
+use saluki_error::{generic_error, GenericError};
+use saluki_event::metric::Metric;
+
+/// Host Tags synchronous transform.
+pub struct HostTagsConfiguration {
+    client: RemoteAgentClient,
+    context_string_interner_bytes: ByteSize,
+    expected_tags_duration: u64,
+}
+
+const DEFAULT_EXPECTED_TAGS_DURATION: u64 = 0;
```

Comment:
Do we actually have this disabled (effectively) by default in the Agent?

Reply:
I believe so. The default of 0 is set [here](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/setup/config.go#L1131) and I don't see it modified elsewhere, the host tags provider just grabs the value [here](https://github.com/DataDog/datadog-agent/blob/35a0d1cec368a8bea71acd55d18efb93136d6f71/pkg/aggregator/host_tag_provider.go#L36)

Reply:
Gotcha, just wanted to be sure. 👍🏻